source/dub/dependency.d

```diff
@@ -21,12 +21,28 @@ import std.exception;
 import std.string;
 import std.sumtype;
 
+/** Encapsulates the name of a package.
+*/
+struct PackageName {
+	string value;
+	@disable this(PackageName name) @safe pure nothrow @nogc;
```

Comment:
Why ?

Reply:
I used it detect unneccesary wrappings `PackageName(PackageName)` because of the alias this. Maybe I should remove the alias instead and add the necessary member (property) member function so PackageName to mimic a string. What do you say?

Reply:
Removing the `alias this` might be good, but not sure how much the code depends on strings at the moment ?

Reply:
I've removed the alias this in favor of more member functions so resolving this.